pkgs/development/compilers/mruby/default.nix

```diff
@@ -1,18 +1,17 @@
 { stdenv, ruby, bison, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
-  name = "mruby-${version}";
-  version = "2.0.0";
+  pname = "mruby";
+  version = "2.0.1";
 
   src = fetchFromGitHub {
     owner   = "mruby";
     repo    = "mruby";
     rev     = version;
-    sha256  = "1r6w1asjshff43ymdwa6xmrkggza99mi2kw88k7ic6ag2j81hcj5";
+    sha256  = "1zm2d5kj9fnfx8ifj8ysrrr838ipwmvz35byzjhprakrg64911p9";
   };
 
   patches = [
```

Comment:
I guess we can remove `patches` and empty lines to save 2 or 3 more SLOC but that's just my OCD speaking :nerd_face:

Reply:
What `patches`? :slightly_smiling_face: